### PR TITLE
TestDuplicateMountpointsForVolumesFromAndMounts: remove unintentional dependency for /tmp/data

### PR DESCRIPTION
**Pull request by AkihiroSuda**
/tmp/data needs to be created before running this test (by some other test)

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

File: integration-cli/docker_cli_volume_test.go [hunk 1]
```diff
@@ -603,6 +604,8 @@ func (s *DockerSuite) TestDuplicateMountpointsForVolumesFromAndMounts(c *check.C
 	c.Assert(strings.TrimSpace(out), checker.Contains, data2)
 
 	// Mounts is available in API
+	err := os.MkdirAll("/tmp/data", 0755)
```
Comment (cpuguy83) [inline integration-cli/docker_cli_volume_test.go]: Need to set a test requirement for`SameHostDaemon` with this

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

File: integration-cli/docker_cli_volume_test.go (hunk 1 above)
Comment (AkihiroSuda) [inline integration-cli/docker_cli_volume_test.go]: done

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
@vdemeester PTAL?

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**